scripts/remove_test_env.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
can we remove the scripts folder since we have test setups and tear down specified in travis.yml file.?

Reply:
removed